.generator/src/generator/templates/rest.j2

```diff
@@ -40,8 +69,14 @@ class RESTClientObject:
         if configuration.assert_hostname is not None:
             addition_pool_args["assert_hostname"] = configuration.assert_hostname
 
-        if configuration.retries is not None:
-            addition_pool_args["retries"] = configuration.retries
+        if configuration.enable_retry == True:
+            retries = DDRetry(
+                total=configuration.max_retries,
+                backoff_factor=configuration.retry_backoff_factor,
+            )
+            addition_pool_args["retries"] = retries
+        else:
+            addition_pool_args["retries"] = False
```

Comment:
```suggestion
```

Reply:
I think we need to keep the false explicit since the request is retried 3 times by default it seems:
[](https://urllib3.readthedocs.io/en/latest/user-guide.html#retrying-requests)

Reply:
Interesting, wonder if it's a new behavior from 3.0. At any rate we should keep the default from previous release, so let's remove it.